db/migrate/20150911195321_add_budget_roles.rb

```diff
@@ -0,0 +1,24 @@
+class AddBudgetRoles < ActiveRecord::Migration
+  def change
+    User.with_email_role_slug!(
```

Comment:
Can we not use the model method in the migration? If the method name changes or goes away, the migration breaks.